### ISSUE TITLE
[ENHANCEMENT] Retain zoom and position across app restarts.

### ISSUE THREAD

**Issue by christophehenry** [michaeltroger/gruenerpass#72]
To mitigate #54, I'm wondering whether the app could retain the zoom level and position across Android lifecycle journey. Currently, the PDFs are rendered full page every time the app comes back to the foreground.

**Comment by michaeltroger** [comment]
Hi, sorry for the late reply.
I just tried to reproduce this.
When bringing the app back to foreground the position and zoom level is correct for me.
With config changes you are maybe referring to e.g. change of language in the Android system though?
Then I have following behavior: The vertical position (active document) is remembered, but the horizontal position (which page within a document) and its zoom level is lost.

**Comment by christophehenry** [comment]
You're right, description is not clear. I meant retain configuration across the whole lifecycle, including app restart.
